### PR TITLE
Updated lint and ts hooks

### PR DESCRIPTION
**Pull request by HardikGoyal2003**
Update typescript tests and linter checks hooks.

### CODE REVIEW

**Comment by HardikGoyal2003** [comment]
@seanlip PTAL! Thanks!

File: TypeScript-tests.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ You can run the TypeScript tests like this:
 
 Python:
 ```console
-python -m scripts.typescript_checks
+python -m scripts.run_typescript_checks --strict_check
```
Comment (seanlip) [inline TypeScript-tests.md]: Are you sure this is correct @HardikGoyal2003? I don't see a strict_check flag in the codebase.

**Review by seanlip — CHANGES_REQUESTED** [review]

File: TypeScript-tests.md (hunk 1 above)
Comment (HardikGoyal2003) [reply TypeScript-tests.md]: @seanlip  you can check this file `` scripts/pre_push_hook.py`` line 75. Thanks!

Comment (seanlip) [reply TypeScript-tests.md]: I did. Your argument is not spelled correctly.

Comment (HardikGoyal2003) [reply TypeScript-tests.md]: @seanlip Typo has been corrected. Thanks!

**Review by seanlip — APPROVED** [review]
Thanks @HardikGoyal2003! LGTM.

**Pull request merged**